app/schemas/RESTAURANTS.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "restaurantId": {
+      "type": "string"
+    },
+    "name": {
+      "type": "string"
+    },
+    "location": {
+      "type": "object",
+      "properties": {
+        "latitude": {
+          "type": "number"
+        },
+        "longitude": {
+          "type": "number"
+        }
+      },
+      "required": ["latitude", "longitude"]
+    },
+    "address": {
```

Comment:
I'd personally be consistent across all collections and pick either `address` or `lat and long`. To determine which is best, I would check out the google maps API, since we're going to need some way of figuring out restaurant proximity relative to the user.

Reply:
I thought having both would be beneficial, address for data storing purposes (eg. this restaurant is located at this st), and lat, long for our map purposes